### ISSUE TITLE
Qualifiers do not get exposed when using @AutoExpose in module

### ISSUE THREAD

**Issue by vpratfr** [lukaspili/Auto-Dagger2#3]
Hi,

Thanks for the library, looks good. While migrating a current project to it, I got stuck with qualifiers: I have an EvenBus instance at the application level and an EvenBus instance at each activity level.

With basic Dagger 2, I deal with that with qualifier annotations such as:

```
/**
 * Qualifier for the module methods to annotate dependencies targeting the activities
 */
@Qualifier
@Documented
@Retention(RetentionPolicy.RUNTIME)
public @interface ForActivity {
}
```

My (manually written) components then have methods like:

```
    @ForActivity
    Context activityContext();

    @ForActivity
    EventBus activityEventBus();
```

When I write the modules using Auto-Dagger2, I specify to auto-expose the class but the qualifier annotation does not get carried over.

```
        @Provides
        @ForApplication
        @AutoExpose(MyApp.class)
        public EventBus provideEventBus() {
            return EventBus.builder().build();
        }
```

That generates a method `EventBus eventBus();` which does not have a qualifier and hence creates a compilation error later on when Dagger2 kicks in.

**Comment by vpratfr** [comment]
I don't know how you do that for @AutoComponent and scopes, but this is basically the same need. Just like you carry the scope annotation over to the generated component, the qualifier annotations should be carried over to the generated exposed methods in the component.

**Comment by lukaspili** [comment]
Thanks for the report. I'll fix that today and add it to the 1.1 release.

**Comment by vpratfr** [comment]
Hi,

Thanks, I started to use Auto-Mortar too, I hope the fix will propagate there too. Libraries look good. 

I am giving a try to Mortar/Flow and I have to say that your power-mortar-flow-dagger2 example project is the best sample out there given the lack of documentation for those projects.

**Comment by lukaspili** [comment]
It's fixed in 1.1 release.
Auto Mortar 1.1 will be released within the hour also.

Also, beware that groupIDs changed for 1.1 releases:
- **com.github.lukaspili.autodagger2**
- **com.github.lukaspili.automortar**
